modules/customizing-component-deployment-resources.adoc

```diff
@@ -0,0 +1,56 @@
+:_module-type: PROCEDURE
+
+[id="customizing-component-deployment-resources_{context}"]
+= Customizing component deployment resources
+
+[role='_abstract']
+You can customize component deployment resources by updating the `.spec.template.spec.containers.resources` section of the YAML file for the component deployment.
+
+.Prerequisites
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+* You are part of the administrator group for {productname-short} in {openshift-platform}.
+
+.Procedure
+. Log in to the {openshift-platform} console as a cluster administrator.
+. In the *Administrator* perspective, click *Workloads* > *Deployments*.
+ifdef::upstream[]
+. From the *Project* drop-down list, select `opendatahub`.
+. In the *Name* column, click the name of the deployment for the component that you want to customize resources for. 
++
+[NOTE]
+====
+For more information on the deployment names for each component, see link:{odhdocshome}/managing-resources/#overview-of-component-resource-customization_custom-resource[Overview of component resource customization].
```

Comment:
For more information **about**

Reply:
Good catch, will update all of these instances.